src/controller/authController.ts

```diff
@@ -0,0 +1,24 @@
+import {Request, Response, Application, response} from "express";
+import { Login } from "../model/auth";
+const authService = require("../service/authService");
+
+module.exports = function(app: Application)
```

Comment:
To make this a bit more user-friendly, could you import/export a more typescript way?

Replace module.exports
```
export const authController => (app: Application) {
     ... code here
}
```
In the app.ts file you can import the controller like this:
`import { authController } from 'path'`

You can then setup the routes by doing:
`authController(app)`